spiceaidocs/docs/reference/sql/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "SQL Reference"
+sidebar_label: "SQL Reference"
+sidebar_position: 4
+pagination_prev: 'reference/index'
+pagination_next: null
+---
+
+
+import DocCardList from '@theme/DocCardList';
```

Comment:
We should note Spice is built on DataFusion and defaults to the PostgreSQL dialect, even when querying federated sources (because we re-write and optimize the SQL).